.github/workflows/pypi-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish to PyPI
+
+on:
+  push:
+
+
+jobs:
+  release:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Setup Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
+
+    - name: Build
+      run: |
+        python3 -m pip install pip wheel build -U
```

Comment:
hatch ?

Reply:
It will get installed in a temp environment automatically by 

    python3 -m build